### PR TITLE
[KOGITO-2947] - Upgrade Operator manifests to the new bundle format

### PR DESCRIPTION
**Pull request by Kaitou786**
See: https://issues.redhat.com/browse/KOGITO-2947

Signed-off-by: Tarun Khandelwal <tarkhand@redhat.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-cloud-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains a link to the JIRA issue
- [x] Pull Request contains a description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,12 +5,10 @@ BUNDLE_IMG ?= quay.io/kiegroup/kogito-cloud-operator-bundle:$(VERSION)
 # Default catalog image tag
 CATALOG_IMG ?= quay.io/kiegroup/kogito-cloud-operator-catalog:$(VERSION)
 # Options for 'bundle-build'
-ifneq ($(origin CHANNELS), undefined)
+CHANNELS=alpha,1.x
 BUNDLE_CHANNELS := --channels=$(CHANNELS)
```
Comment (radtriste) [inline Makefile:9]: so we will always define the `BUNDLE_CHANNELS` and `BUNDLE_DEFAULT_CHANNEL` right ?

Comment (radtriste) [reply Makefile:9]: should those values be always the same as in `annotations.yaml` and `bundle.Dockerfile` ?

Comment (Kaitou786) [reply Makefile:9]: annotation.yaml and bundle.Dockerfile are autogenerated files, defining these values edit those two files

Comment (Kaitou786) [reply Makefile:9]: I defined the same channels we currently have in the OLM, and was present in the package.yml, we can review later about what channels we should offer

**Review by radtriste — APPROVED** [review]

**Pull request merged**